### DISCUSSION TITLE
tinyflux fills up temp directory

### DISCUSSION THREAD

**Discussion by lwallimann** [citrusvanilla/tinyflux#43] [Q&A]
My script runs every minute and each time tinyflux create a temp file. In my case (windows) under AppData/loal/temp. I found that tinyflux uses the tempfile Python module. Is there a flag so that tinyflux does not use the temp directory?

thanks a lot

**Comment by citrusvanilla** [comment]
@lwallimann The only time tinyflux should create a temporary file is when you are using the CSVStorage instance, and performing any operation that removes data, which includes dropping measurement(s), updating individual Points (or all Points), and removing individual Points (or all Points).  Are you performing this every minute?

**Comment by lwallimann** [comment]
@citrusvanilla yes I perfom this every minute (updating and removing). I found a solution by deleting it at the end of the skript with the tempfile python module.
